client/galaxy/scripts/components/Masthead/MastheadItem.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+    <b-nav-item
+        v-if="!menu"
+        :class="classes"
+        :style="styles"
+        :active="!tab.disabled"
+        :id="tab.id"
+        :href="formatUrl(tab.url)"
+        :target="tab.target || '_parent'"
+        role="menuitem"
+        :link-classes="linkClasses"
+        v-b-tooltip.hover.bottom
+        :title="tab.tooltip"
+        v-b-popover.manual.bottom="{ id: tab.id, content: popoverNote, html: true }"
+        @click="open(tab, $event)"
+    >
+        <template v-if="tab.icon">
```

Comment:
It'd be great if future work could keep in mind and use the new(ish) vue-fontawesome work that was introduced to swap out the old style fontawesome classes and includes.